test/integration/test_io_raw_terminal.py

```diff
@@ -13,11 +13,18 @@
 
 from moler.cmd.unix.ls import Ls
 from moler.cmd.unix.ping import Ping
+from moler.cmd.unix.gunzip import Gunzip
 from moler.cmd.unix.whoami import Whoami
 from moler.exceptions import CommandTimeout
 from moler.io.raw.terminal import ThreadedTerminal
 
 
+def test_terminal_cmd_gunzip(terminal_connection):
```

Comment:
Do not change integration test!
Please do not commit test on real connection into repository!

Reply:
ok